test/e2e/dra/dra.go

```diff
@@ -136,6 +136,30 @@ var _ = ginkgo.Describe("[sig-node] DRA [Feature:DynamicResourceAllocation]", fu
 				}
 			})
 
+			ginkgo.It("supports external claim referenced by multiple containers of multiple pods", func() {
```

Comment:
Can you add another variant of this with immediate binding?

Reply:
Actually, it probably makes sense to have several of these tests in both variants. Not all combinations make sense, but all of them are possible and thus should be tested.

This can be done with two `Context` calls that call a helper function with a parameter that determines who the build creates the claim specs.

Reply:
Should it be a third level of Context? How to determine which test cases should be run and which shouldn't in certain mode?

Reply:
done. Not sure I did it the way you suggested, but it seems to work as expected:
```
KUBECONFIG=/var/run/kubernetes/admin.kubeconfig go test -v ./test/e2e -args -test.v -ginkgo.focus=Feature:DynamicResourceAllocation
...
Ran 12 of 7079 Specs in 392.284 seconds
SUCCESS! -- 12 Passed | 0 Failed | 0 Pending | 7067 Skipped
```